src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -109,16 +109,13 @@ public function getAction($webspace, $id, Request $request)
     /**
      * Create a new custom-url object.
      *
-     * @param string $webspace
```

Comment:
would not remove this

Reply:
In other function in this file the parameters are also descriped like this. And when we want to go up to higher PHPStan levels or to PHP 8 we also have to do this

Reply:
the problem is not that the change would be wrong, the problem is that we cannot break backwards compatibility of this class. if somebody has overwritten/extended the class, his project will break if we change the typehints of the methods. therefore we need to use phpdoc types instead of typehints

Reply:
> And when we want to go up to higher PHPStan levels or to PHP 8 we also have to do this

I really doubt this. Can you provide any source for this?